berrynet/client/camera.py

```diff
@@ -1,153 +1,150 @@
 #!/usr/bin/python3
-#
-# Copyright 2018 DT42
-#
-# This file is part of BerryNet.
-#
-# BerryNet is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# BerryNet is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with BerryNet.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Copyright 2018 DT42
 
-import argparse
-import logging
-import time
+This file is part of BerryNet.
+
+BerryNet is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+BerryNet is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with BerryNet.  If not, see <http://www.gnu.org/licenses/>.
+"""
 
+import time
+import logging
+import argparse
 from datetime import datetime
 
 import cv2
 
 from berrynet import logger
-from berrynet.comm import Communicator
-from berrynet.comm import payload
+from berrynet.comm import Communicator, payload
 
 
 def parse_args():
-    ap = argparse.ArgumentParser()
-    ap.add_argument(
-        '--mode',
-        default='stream',
-        help='Camera creates frame(s) from stream or file. (default: stream)'
+    """
+    args
+    """
+    arg_parser = argparse.ArgumentParser()
+    arg_parser.add_argument(
+        "--mode",
+        default="stream",
+        help="Camera creates frame(s) from stream or file. (default: stream)",
     )
-    ap.add_argument(
-        '--stream-src',
```

Comment:
As described in PEP8:

> In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. 

We choose single quote because most of the Python official docs use single quote.

Example: https://docs.python.org/3/tutorial/controlflow.html#for-statements

---
However, we are open to adopt a new rule if there is a good reason behind. It will be great if you can share more about your opinions with me. :smiley: 

https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python